pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -227,7 +227,8 @@ protected void internalDeleteNamespace(AsyncResponse asyncResponse, boolean auth
         boolean isEmpty;
         List<String> topics;
         try {
-            topics = pulsar().getNamespaceService().getListOfPersistentTopics(namespaceName).join();
+            topics = pulsar().getNamespaceService().getListOfPersistentTopics(namespaceName)
```

Comment:
nice catch!
There are many places which call `getListOfPersistentTopics` also use `join` to get the CompletableFuture result, would you please fix them? thanks.

Reply:
Ok, I'll start a new PR and fix all the places where join is used